### ISSUE TITLE
PMO: Unable to run documentjs on Windows

### ISSUE THREAD

**Issue by matthewp** [donejs/donejs#478]
```
donejs : C:\Users\Matthew\dev\place-my-order\node_modules\q\q.js:126
At line:1 char:1
+ donejs document 2> err
+ ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (C:\Users\Matthe...ules\q\q.js:126:S 
   tring) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError

                    throw e;
                    ^

Error: ENOENT: no such file or directory, open 
'C:\Users\Matthew\dev\place-my-order\docs\place-my-order|restaurant|list.html'
    at Error (native)



npm

ERR!
 Windows_NT 10.0.10586

npm

ERR!

argv
 "C:\\Program Files\\nodejs\\node.exe" "C:\\Program 
Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "document"

npm

ERR!

node
 v5.3.0

npm

ERR!

npm 
 v3.3.12

npm

ERR!

code
 ELIFECYCLE

npm

ERR!
 place-my-order@0.0.0 document: `documentjs`

npm

ERR!
 Exit status 1

npm

ERR!


npm

ERR!
 Failed at the place-my-order@0.0.0 document script 'documentjs'.

npm

ERR!
 Make sure you have the latest version of node.js and npm installed.

npm

ERR!
 If you do, this is most likely a problem with the place-my-order package,

npm

ERR!
 not with npm itself.

npm

ERR!
 Tell the author that this fails on your system:

npm

ERR!
     documentjs

npm

ERR!
 You can get their info via:

npm

ERR!
     npm owner ls place-my-order

npm

ERR!
 There is likely additional logging output above.



npm

ERR!
 Please include the following file with any support request:

npm

ERR!
     C:\Users\Matthew\dev\place-my-order\npm-debug.log

Error: Command `npm` did not complete successfully
    at ChildProcess.<anonymous> 
(C:\Users\Matthew\dev\place-my-order\node_modules\donejs-cli\lib\utils.js:37:23)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at ChildProcess.cp.emit (C:\Users\Matthew\dev\place-my-order\node_modules\cro
ss-spawn-async\lib\enoent.js:37:29)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
```

**Comment by matthewp** [comment]
This is probably the underlying issue: https://github.com/bitovi/documentjs/issues/216

**Comment by matthewp** [comment]
Replacing the | with __ does fix the issue.

**Comment by matthewp** [comment]
Fixed by DocumentJS 0.4.2
